src/Main.purs

```diff
@@ -92,7 +92,7 @@ main = launchAffVoid do
             Process.exit 1
       liftEff do
         runEffFn2 gaze
-          (concatMap (fileGlob directory) sourceDirectories)
```

Comment:
You'll also want to replace the binder for `directory` with `_` instead.

Reply:
It's used in `        log ("Watching " <> directory <> " on port " <> show port')`

Reply:
Oh I see, nevermind then!